pkg/operator/starter.go

```diff
@@ -110,6 +111,9 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 		controllerContext.EventRecorder,
 	)
 
+	//register metrics
```

Comment:
Please omit comments that provide no info over and above the code.

Reply:
removed it now.